glide.lock

```diff
@@ -103,36 +103,34 @@ imports:
   subpackages:
   - capnslog
 - name: github.com/couchbase/vellum
-  version: eb6ae3743b3f300f2136f83ca78c08cc071edbd4
-  subpackages:
-  - utf8
-- name: github.com/couchbaselabs/vellum
-  version: eb6ae3743b3f300f2136f83ca78c08cc071edbd4
+  version: 1782e2d46ce2a9f28bd1ac1b46cd141c0d887c9b
+  repo: https://github.com/m3db/vellum
   subpackages:
   - regexp
+  - utf8
 - name: github.com/davecgh/go-spew
   version: 5215b55f46b2b919f50a1df0eaa5886afe4e3b3d
   subpackages:
   - spew
 - name: github.com/dgrijalva/jwt-go
   version: d2709f9f1f31ebcda9651b03077758c1f3a0018c
 - name: github.com/dgryski/go-bits
-  version: 2ad8d707cc05b1815ce6ff2543bb5e8d8f9298ef
+  version: bd8a69a71dc203aa976f9d918b428db9ac605f57
 - name: github.com/edsrzf/mmap-go
   version: 0bce6a6887123b67a60366d2c9fe2dfb74289d2e
 - name: github.com/ghodss/yaml
   version: 0ca9ea5df5451ffdf184b4428c902747c2c11cd7
 - name: github.com/glycerine/go-unsnap-stream
-  version: 62a9a9eb44fd8932157b1a8ace2149eff5971af6
+  version: 9f0cb55181dd3a0a4c168d3dbc72d4aca4853126
```

Comment:
nit: If we don't have to update a bunch of this stuff maybe we shouldn't?  Although if it builds maybe not as important.

Reply:
yeah fair, i'll revert.

Reply:
done